include/entity.h

```diff
@@ -2152,6 +2152,25 @@ typedef struct {
     /* 0xAD */ u8 unkAD;
 } ET_Diplocephalus;
 
+typedef struct {
+    /* 0x7C */ struct Entity* parent;
+    /* 0x80 */ struct Entity* entity;
+    /* 0x84 */ s16 : 16;
+    /* 0x86 */ s16 : 16;
+    /* 0x88 */ s32 : 32;
+    /* 0x8C */ s32 : 32;
+    /* 0x90 */ struct Entity* unk90;
+    /* 0x94 */ s32 : 32;
+    /* 0x98 */ s32 velocityY;
+    /* 0x9C */ s16 : 16;
+    /* 0x9E */ u8 unk9E;
+    /* 0x9F */ u8 : 8;
+    /* 0xA0 */ u32 : 32;
+    /* 0xA4 */ s32 posX;
+    /* 0xA8 */ s32 posY;
+    /* 0xAC */ s32 unkAC;
```

Comment:
`unkAC` here seems to be an `s32` not a `u8` like it is in the other extension, otherwise these seem to be pretty similar.